docs/circle.rst

```diff
@@ -0,0 +1,108 @@
+==================
+:mod:`pygame_geometry.Circle`
+==================
+
+.. currentmodule:: pygame_geometry
+
+.. class:: Circle
+
+    | :sl:`pygame object for representing a circle`
+    | :sg:`Circle((x, y), radius) -> Circle`
+    | :sg:`Circle(x, y, radius) -> Circle`
+
+    The `Circle` class provides many useful methods for collision / transform and intersection.
+    A `Circle` can be created from a combination of a pair of coordinates (x, y) that represents
+    the center of the circle and a radius. Circles can also be created from python objects that
+    are already a `Circle` or have an attribute named "circle".
+
+    Any function that requires a Circle argument also accepts any of these
+    values as Circles, the following are all valid ways:
+    ::
+        ((x, y), radius)
+        (x, y, radius)
+
+    The Circle functions that change its position or size return a new copy of the
```

Comment:
I feel this might be a bit clearer: "The circle functions which modify the position or size... etc"